app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanelMain.tsx

```diff
@@ -12,7 +13,8 @@ export const Panel = ({
 	children: ReactNode;
 	testId?: string;
 }) => (
-	<div className={cx(connectionPanelMainTabs)} data-testid={testId}>
+	<div className={cx(testId === 'setUpGuidePanel'
+		? setUpGuideContainer : connectionPanelMainTabs)} data-testid={testId}>
```

Comment:
/nit attribute names case descrepenciesssssss, it s case-insensitive but nice to have....   we have a mixture of `data-testid=` and `testId=`